server/routes/clientData.js

```diff
@@ -0,0 +1,67 @@
+const debug = require('debug')('perf-widget:lib:clientData');
+const dataFor = require('../lib/dataFor');
+/*const dataFor = function (){
```

Comment:
Sure, I just wanted to make sure it was somewhere in the commit history while we were still putting bits together